newIDE/app/src/UI/Grid.js

```diff
@@ -36,6 +36,7 @@ export const Column = props => (
       alignItems: props.alignItems || 'stretch',
       justifyContent: props.justifyContent,
       flex: props.expand ? 1 : undefined,
+      maxHeight: props.maxHeight ? props.maxHeight : undefined,
```

Comment:
`Column` is a component used a lot for the layout, and as such the less flexibility is has in its prop the better. This ensure consistency in the interface, and avoid bloating it with tons of options. There should be only one way to construct an interface with `Column` (and `Line`).

As such I would favor having a prop called "useMaxHeight" that is setting `maxHeight` to 100% if true, or undefined otherwise.
This will prevent anyone from being tempted to use `maxHeight="95%"` to "add some margins" or some other anti pattern (the proper pattern being to use a `Line` if you need margins or to stack elements).